boards/nxp/rd_rw612_bga/rd_rw612_bga.dtsi

```diff
@@ -83,6 +83,7 @@ arduino_i2c: &flexcomm2 {
 	ahb-cacheable;
 	ahb-read-addr-opt;
 	rx-clock-source = <1>;
+	rx-clock-source-b = <3>;
```

Comment:
Is this change to the `flexcomm` node required?

Reply:
Yes- the IS66WVQ8M4 PSRAM uses a different clock source to send data (specifically the DQS pad of the flash chip). This is different from the mx25u51245g flash on port A, which uses a loopback clock from the DQS pad)